### ISSUE TITLE
Teamredminer vs lolminer on solo kaspa. 

### ISSUE THREAD

**Issue by CatStoleMyRvnCoin** [todxx/teamredminer#676]
I'm looking if anyone else has had similar issues. 

Just a random observation about the two and I'm not sure if it's lack of luck or it's a problem with teamredminer and solo kaspa, but I did a 12 hour each test on both.

Lolminer ping to woolypooly Canada solo shares are 2-3ms and netted me a couple blocks.

Teamredminer the miner I prefer to use, same pool/server had 6ms shares and higher hashrate but actually netted no blocks and on teamredminer before I switched back to lolminer I seen effort hit 433% before I said enough and switched it.

Now the second test everything the same again pool server wise. 
3 blocks on lolminer, seems to work normal and normal on lolminer is probably 2-5 blocks in 12 hours for my set up. 

And for the second test on teamredminer no blocks and effort hit 363% not the full 12 hours before I said enough and switched back to lolminer again.

Again to be clear I prefer team red because it just seems more solid for hashrate and using less power. So maybe it's a issue with solo kaspa since it was just put out?

**Comment by CamJava007** [comment]
I have noticed this too.  Pool mining is fine, but solo finds significantly fewer blocks than LOL.  Team Red is also my usual program of choice, for the same reasons that you've mentioned.

**Comment by CatStoleMyRvnCoin** [comment]
> I have noticed this too. Pool mining is fine, but solo finds significantly fewer blocks than LOL. Team Red is also my usual program of choice, for the same reasons that you've mentioned. I'm on Windows. I find, for the K-HeavyHash Algo, windows uses a good deal less electricity than Linux... which surprised me, because it was opposite with ETHASH.

Yeah not sure what is actually going on, I've done another test again since.

24 hours team red and put only 7gh on solo kaspa teamredminer and got 1.block.

Then switched same amount onto lolminer for same time and cam out with 4.blocks.

I think something is wrong with team red solo 🤷‍♂️

I definitely wouldn't say it's anything other then software error maybe because nothing else leads me to believe somehow blocks could be taken, but I'm not technical like that so I don't know.

**Comment by CamJava007** [comment]
The benefit to using Team Red Pool... is that you can Dual Mine Zil without increasing your electric cost.  The Team Red code does not require your primary algo to be ETHASH for Zil mining as others do... it can literally be anything.
